components/Comment/CommentEditor.tsx

```diff
@@ -74,7 +103,12 @@ const CommentEditor = ({
         )}
         <div className={css(styles.editor)}>
           <div ref={quillRef} />
-          <div className={css(styles.toolbarContainer)}>
+          <div
+            className={css(
+              styles.toolbarContainer,
+              isPreviewMode && styles.hidden
```

Comment:
Instead of hiding, with display hidden, should we instead conditionally render this section?

```
isPreviewMode && ...
```

Reply:
That was my initial thought but conditionally rendering this results in a choppy quill toolbar rendering when `isPreviewMode == false`